src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdatesReporter.groovy

```diff
@@ -144,7 +150,21 @@ class DependencyUpdatesReporter {
       buildDependenciesGroup(current),
       buildDependenciesGroup(outdated),
       buildDependenciesGroup(exceeded),
-      buildDependenciesGroup(unresolved)
+      buildDependenciesGroup(unresolved),
+      buildGradleUpdateResults()
+    )
+  }
+
+  /**
+   * Create a {@link  GradleUpdateResults} object from the information provided by the {@link GradleUpdateChecker}
+   * @return filled out object instance
+   */
+  private GradleUpdateResults buildGradleUpdateResults() {
+    return new GradleUpdateResults(
+      running: new GradleUpdateResult(gradleUpdateChecker.runningGradleVersion, gradleUpdateChecker.runningGradleVersion),
```

Comment:
What's the reason behind running here?

Reply:
It is used to output the update path `4.6 -> 4.7` (with 4.6 being the 'running' version of gradle). It's also exported in the xml/json report and can then be programmatically evaluated.